plugins/dependencies/src/main/kotlin/dependencies/Kodein.kt

```diff
@@ -0,0 +1,69 @@
+@file:Suppress("PackageDirectoryMismatch", "SpellCheckingInspection", "unused")
+
+import org.gradle.api.Incubating
+import org.gradle.kotlin.dsl.IsNotADependency
+
+@Incubating
+object Kodein {
+
+
+    /**
+     * painless Kotlin dependency injection
+     *
+     * - [Official website here](https://kodein.org/di/)
+     * - GitHub page: [Kodein-Framework/Kodein-DI](https://github.com/Kodein-Framework/Kodein-DI)
+     * - [GitHub Releases here](https://github.com/Kodein-Framework/Kodein-DI/releases)
+     */
+    val di = DI(usePlatformConstraints = true)
+    val `di-no-bom` = DI(usePlatformConstraints = false)
```

Comment:
This is one of the multiple ways to organise/namespace the no BoM version (and this one is inconsistent with what we have in `Firebase`).

We'll have to discuss this and settle on a convention that we can then mention in the contributing guide.